spyder/widgets/panels/edgeline.py

```diff
@@ -49,11 +48,26 @@ def set_enabled(self, state):
         self._enabled = state
         self.setVisible(state)
 
-    def set_column(self, column, index=0):
-        """Set edge line column value"""
-        self.columns[index] = column
+    def set_columns(self, columns):
+        """Set edge line columns values."""
+        if isinstance(columns, list):
+            self.columns = columns
+        elif isinstance(columns, int):
+            self.columns = [columns]
+        elif isinstance(columns, str):
+            try:
+                self.columns = [int(e) for e in columns.split(',')]
+            except:
+                #TODO Log error, invalid columns argument
+                pass
```

Comment:
@rlaverde its fine, maybe just add a comment to say that Configuration of a text box for now does not assume anything about the content so that it is clear why this is needed, ok :-) ?